src/DeserializationPojo/ListUsersResponsePojo.java

```diff
@@ -0,0 +1,51 @@
+package DeserializationPojo;
+
+import java.util.List;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ListUsersResponsePojo {
+	private int page;
+	@JsonProperty("per_page")
+	private int perPage;
+	private int total;
+	@JsonProperty("total_pages")
+	private int totalPages;
+	private List<UserPojo> data;
+	private AdPojo ad;
+	public int getPage() {
```

Comment:
It is generally a good idea to keep a one line gap between properties and methods. Help visualize two different groups of things.